### PR TITLE
Add reactions to meetings. Rewrote meeting from class component -> hook.

### PR DESCRIPTION
**Pull request by jonasdeluna**
image

### CODE REVIEW

**Comment by ivarnakken** [comment]
Cool! 💯 We can use this to register who wants pizza for our meetings haha! 

Thought; could we perhaps move it down to the comment section? I feel like the sidebar is only for general info + admin stuff.

**Comment by jonasdeluna** [comment]
> 
Can try this :+1:

**Comment by jonasdeluna** [comment]
image
Like this?

File: app/routes/meetings/MeetingDetailRoute.tsx [hunk 1]
```diff
@@ -63,12 +70,16 @@ const mapStateToProps = (state, props) => {
...
+  const emojis = selectEmojis(state);
   return {
     meetingsToken,
     user: props.currentUser,
     showAnswer,
     meeting,
     currentUser,
+    emojis,
+    fetchingEmojis: state.emojis.fetching,
+    fetching: state.meetings.fetching,
```
Comment (ivarnakken) [inline app/routes/meetings/MeetingDetailRoute.tsx:82]: This seems to be unused.

**Review by ivarnakken — APPROVED** [review]
Awesome! 🍕

**Comment by jonasdeluna** [comment]
^removed & placed emojis at the bottom
```
fetchingEmojis: state.emojis.fetching,
fetching: state.meetings.fetching,
```

File: app/routes/meetings/MeetingDetailRoute.tsx [hunk 2]
```diff
@@ -22,7 +26,10 @@ const loadMeeting = (
     },
   },
   dispatch
-) => (loggedIn ? dispatch(fetchMeeting(meetingId)) : Promise.resolve());
+) =>
+  loggedIn
+    ? dispatch(fetchMeeting(meetingId), fetchEmojis())
```
Comment (LudvigHz) [inline app/routes/meetings/MeetingDetailRoute.tsx:31]: Are you sure you can dispatch like this? According to the docs, this is not an option.
You could rather do a `Promise.all([dispatch(fetchmeeting(...)), dispatch(fetchemojis())]`

Comment (LudvigHz) [reply app/routes/meetings/MeetingDetailRoute.tsx:31]: Also, why are you fetching emojis here and passing `fetchEmojis` into the reaction picker?

File: app/routes/meetings/MeetingDetailRoute.tsx (hunk 1 above)
Comment (ivarnakken) [reply app/routes/meetings/MeetingDetailRoute.tsx]: @jonasdeluna You removed `fetchingEmojis`, but that's actually used. I was only referring to `fetching`. Sorry for not being more specific 😅

File: app/routes/meetings/MeetingDetailRoute.tsx (hunk 2 above)
Comment (jonasdeluna) [reply app/routes/meetings/MeetingDetailRoute.tsx:31]: Hmm I'm not entirely sure actually. It was written that way in the ArticleDetailRoute.tsx file, but I'll try the promise.all instead.

**Comment by jonasdeluna** [comment]
image

Changed placement back to the sidebar (?)

**Comment by ivarnakken** [comment]
> I also liked having the reactions someplace more visible than under the content, as I feel they "disappear" if you have just a little text in the meeting

@LudvigHz 
The same goes for comments though. I feel like "interactions" like comments and reactions should be grouped together? Because that's pretty common. Besides, I feel like comments are more important than reactions, and we've put that at the bottom.

**Comment by LudvigHz** [comment]
> The same goes for comments though. I feel like "interactions" like comments and reactions should be grouped together? Because that's pretty common. Besides, I feel like comments are more important than reactions, and we've put that at the bottom.

Yeah, I agree. And it is a bit annoying sometimes that comments are all the way at the bottom, but on the other hand, comments don't really fit anywhere else, whereas reactions can fit in a small space almost anywhere, so it's more flexible where we can put it

**Comment by jonasdeluna** [comment]
Could not reproduce the cypress fail, but may be due to lack of the correct backend version?

File: cypress/support/component.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+// ***********************************************************
```
Comment (LudvigHz) [inline cypress/support/component.js:1]: Remove these files and we're good to go :upside_down_face:

**Review by LudvigHz — CHANGES_REQUESTED** [review]

File: cypress/support/component.js (hunk 3 above)
Comment (jonasdeluna) [reply cypress/support/component.js]: Hopefully I removed the correct files.

File: app/routes/pages/components/subcomponents/Statistic/Statistic.css [hunk 4]
```diff
@@ -9,7 +9,7 @@
 }
 
 .statistic {
-  font-size: 2.2rem;
+  font-size: 1.2rem;
```
Comment (LudvigHz) [inline app/routes/pages/components/subcomponents/Statistic/Statistic.css:12]: What is this?

File: app/routes/pages/components/LandingPage.css [hunk 5]
```diff
@@ -56,7 +56,7 @@ div > .locationContainerItem {
 
 .contactTitle {
   color: var(--lego-font-color);
-  font-size: 1.7rem;
+  font-size: 1rem;
```
Comment (LudvigHz) [inline app/routes/pages/components/LandingPage.css:59]: What does this do and why is it part of this PR?

**Review by LudvigHz — APPROVED** [review]

**Pull request merged**